src/tests/ef_vi/eflatency.c

```diff
@@ -130,6 +132,13 @@ static void init_udp_pkt(void* pkt_buf, int paylen)
   memcpy(eth->ether_dhost, remote_mac, sizeof(remote_mac));
   ef_vi_get_mac(&rx_vi.vi, driver_handle, eth->ether_shost);
   eth->ether_type = htons(0x0800);
```

Comment:
I think I would prefer if this was moved into an `else` under the `if( cfg_vlan )` check. I understand that this is inoffensive because you correctly write this data region, but I would prefer not to incorrectly write to `eth_vlan->ether_vtype` here if we are using a VLAN.